gap/ToolFunctions.gi

```diff
@@ -78,7 +78,15 @@ InstallGlobalFunction( AutoDoc_WriteDocEntry,
     od;
     AppendTo( filestream, ">\n" );
 
-    # Function heades
+    # Next possibly the heading for the entry
+    if Length( maybe_heading ) > 0 then
+        maybe_heading := maybe_heading[ 1 ];
+        if IsString( maybe_heading ) then
+            AppendTo( filestream, "<Heading>", maybe_heading, "</Heading>\n" );
+        fi;
+    fi;
```

Comment:
With the change to `heading`, this could be replaced by
```gap
if IsString( heading ) then
  AppendTo( filestream, "<Heading>", heading, "</Heading>\n" );
fi;
```